functests-performance-profile-creator/1_performance-profile_creator/ppc.go

```diff
@@ -108,7 +109,103 @@ var _ = Describe("[rfe_id:OCP-38968][ppc] Performance Profile Creator", func() {
 			Expect(cInfo).To(BeEquivalentTo(expectedInfo), "regression test failed for '%s' case", expectedClusterInfoPath)
 		}
 	})
-
+	Context("Systems with Hyperthreading enabled", func() {
+		It("[test_id:41419] Verify PPC script fails when reserved cpu count is 2 and requires to split across numa nodes", func() {
+			Expect(ppcPath).To(BeAnExistingFile())
+			mustGatherFullPath := path.Join(mustGatherPath, "must-gather.bare-metal")
+			Expect(mustGatherFullPath).To(BeADirectory())
+			cmdArgs := []string{
+				fmt.Sprintf("--disable-ht=%t", false),
```

Comment:
a few of these can be constant string, e.g (untested)
```
cmdArgs := []string{
   "--disable-ht=false"
   ...
```

Not worth a reupload just for this though